### PR TITLE
Add PortProxy artifact to windows.yaml

### PR DESCRIPTION
**Pull request by Karneades**

### CODE REVIEW

File: data/windows.yaml [hunk 1]
```diff
@@ -1563,6 +1563,19 @@ urls:
 - 'https://support.microsoft.com/en-us/kb/323525'
 - 'https://blog.malwarebytes.com/detections/pum-optional-disallowrun/'
 ---
+name: WindowsPortProxyConfiguration
+doc: Windows PortProxy registry keys (set by netsh portproxy command or manually).
+sources:
+- type: REGISTRY_KEY
+  attributes:
+    keys:
+    - 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PortProxy\v4tov4\tcp\*'
```
Comment (joachimmetz) [inline data/windows.yaml:1572]: Would it make sense to add support for `v4tov6`, `v6tov4`, and `v6tov6` sub keys as well ? And maybe different protocols such as `udp` that are suported by `netsh`?

**Comment by joachimmetz** [comment]
@Karneades thx for the proposed changes, I'm wondering if we can/should make this definition more generic.

File: data/windows.yaml (hunk 1 above)
Comment (Karneades) [reply data/windows.yaml:1572]: Yes, think that too to make the defintion more generic. No other keys/values are set there expect these port forwardings on my test machine.

**Comment by Karneades** [comment]
Using ** results in having an entry also for the keys themselves. I use no the more specific version with normal wildcards.

**Comment by Karneades** [comment]
Are you planning to merge the artifact?

**Comment by joachimmetz** [comment]
I'll have another look when time permits. Thx for the reminder it likely fell through the crack since the PR was not marked as to review.

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**